src/ripple/app/misc/Validations.cpp

```diff
@@ -206,12 +206,12 @@ class ValidationsImp : public Validations
 
         auto const now = app_.timeKeeper().closeTime();
         auto const signTime = val->getSignTime();
-
+        auto const p = ConsensusParms{};
```

Comment:
It won't be compile time constant there.  Instead `Validations` has its own parameters that it takes in its constructor which will allow simulations to vary those values too.